src/helpers/utils.ts

```diff
@@ -252,14 +252,17 @@ async function updateWalletConnectWhitelist(
 }
 
 export function getSpaceController(space: string, network = NETWORK) {
-  const shibariumNetworkMap = {
-    mainnet: '109',
-    testnet: '157'
+  const tld = space.split('.').slice(-1)[0];
```

Comment:
Not too important, but this can cause trouble in future if some network in `getSpaceController` doesn't check `endsWith` (we are checking in `getShibariumNameOwner` and `getUDNameOwner` but not `getEnsSpaceController`)


Because 
`"sonic".split('.').slice(-1)[0];` will return `sonic`

It is fine now, but may cause confusion it future because the tldmapping doesn't contain any `.`  but not on snapshot.js https://github.com/snapshot-labs/snapshot.js/blob/master/src/utils.ts#L48-L57

Reply:
it will then default to null, and fallback to `DEFAULT_NETWORK`

Reply:
hmm no `sonic` as space name will still go to sonic tld